knora/dsplib/utils/xml_upload.py

```diff
@@ -189,6 +191,77 @@ def _convert_ark_v0_to_resource_iri(ark: str) -> str:
     return "http://rdfh.ch/" + project_id + "/" + dsp_uuid
 
 
+def _configure_logger(server: str, shortcode: str, default_ontology: str, verbose: bool) -> dict[str, str]:
+    """
+    Creates a directory structure where the logs will be saved at, depending on the OS, the server, the project, and
+    the ontology. Creates also a logger, depending on the verbosity, and stores it as a module variable in xml_upload.py
+    and in shared.py.
+
+    Args:
+        server: server address where the data is uploaded to
+        shortcode: shortcode of the project
+        default_ontology: name of the ontology
+        verbose: verbose switch
```

Comment:
The "verbose" option is something that is independent of the logging. Logs are dependent on the loglevel. The "verbose" flag can be set by the user to controle console output. The two things should not be mixed up in my opinion.